### DISCUSSION TITLE
The maize model yield prediction function

### DISCUSSION THREAD

**Discussion by wxystudio** [APSIMInitiative/ApsimX#8792] [General]
Hello I'm studying the maize yield prediction. I want to know how the function works in the APSIM model. I see the APSIM paper "APSIM e Evolution towards a new generation of agricultural systems simulation" and find it is related to two papers:
<img width="377" alt="image" src="https://github.com/APSIMInitiative/ApsimX/assets/17384283/724fb863-3ad9-45b9-b861-62cc779f7075">
 But when I turn to these two papers I find there is no detailed description. Could you tell me: 1) where is the maize model function in this code, and 2) where can I find the details of simulation?

**Comment by hol353** [comment]
Unfortunately, there is no maize model description paper for APSIM. There is some documentation here: https://builds.apsim.info/api/nextgen/docs/Maize.pdf. 

Looking at the code will not help you. The APSIM plant models are built using the PMF (https://www.sciencedirect.com/science/article/pii/S1364815214002588). This means the code is generic to all plant models and is split into 100s of functions.

In the user interface you can open the example maize simulation, right click on the maize model and choose 'Show Model Structure'. This will show you how the maize model is structured and parameterised. If you study this and read the documentation and the PMF paper (links above) you might understand more about how the maize model functions.
